src/app/results/page.tsx

```diff
@@ -39,23 +40,19 @@ export default function Results() {
         </div>
         <div className={styles.topRowColumn}>GENDER AGE</div>
       </div>
+      <div className={styles.contentRow}>        
+      <h2 className={styles.sectionsHeadings}>Main Summary</h2>
+        <MainSummarySection dataKey={dataKey}></MainSummarySection>
```

Comment:
With React components you can close your tags in the initial tag. It makes the code a little shorter and would look like this: `<MainSummarySection dataKey={dateKey} />.